sr_hand_config/common/src/sr_hand_config/load_diagnostic_analyzer.py

```diff
@@ -75,13 +74,12 @@ def _modify_analyzers_for_correct_index(self, analyzer, hand_serial):
 
             analyzer['analyzers']['shadow_hand']['analyzers'][individual_analyzer]['regex'] = new_regex
 
-    def _side_to_prefix(self, side):
+    def _side_to_prefix(self, side):  # pylint: disable=R0201
```

Comment:
Instead of disabling this, maybe we should move function outside scope of the class?
'self' is not necessary

Reply:
better to make it as staticmethod